test/Microsoft.IdentityModel.Tests/Microsoft.IdentityModel.Tests.csproj

```diff
@@ -13,6 +13,7 @@
     <SignAssembly>true</SignAssembly>
     <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(TestTargets);net45</TargetFrameworks>
     <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netcoreapp2.0</TargetFrameworks>
+    <IsTestProject>false</IsTestProject>
```

Comment:
>false [](start = 19, length = 5)

?

Reply:
This project doesn't contain any test methods.
It's a utility test project which contains test stubs, mocks, reference data, utilities, etc.

Reply:
imo, then the name of the project is misleading.

Reply:
@henrik-me 	I agree. Perhaps we should rename it to m.im.testutilities.
Can we open a new issue?

Reply:
I don't mind renaming it as part of this PR. _TestUtilities_ looks fine to me.

Reply:
On second thought, we have a static [class](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/dev/test/Microsoft.IdentityModel.Tests/TestUtilities.cs#L51) named _TestUtilities_, which is used in many places.
Introducing the _Microsoft.IdentityModel.TestUtilities_ namespace would clash with a method call from the _TestUtilities_ class e.g. `TestUtilities.AssertFailIfErrors()` and doing something like `TestUtilities.TestUtilities.XYZ()` looks ugly to me.

I would suggest renaming the project to _m.im.TestUtils_ or just _m.im.Utilities_.